web/src/components/storage/ProposalSpacePolicySection.jsx

```diff
@@ -447,7 +447,12 @@ export default function ProposalSpacePolicySection({
   const currentPolicy = SPACE_POLICIES.find(p => p.id === settings.spacePolicy) || SPACE_POLICIES[0];
 
   return (
-    <Section title={_("Find Space")}>
+    <Section
+      // TRANSLATORS: The storage "Find Space" section's title
+      title={_("Find Space")}
+      // TRANSLATORS: The storage "Find space" sections's description
+      description={_("Allocating the file systems might need to find free space in the devices listed below. Choose how to do it")}
```

Comment:
NP: Do we have any rule for limiting the line length?

Reply:
Something that I myself asked in the past. We should have both, 

* a maximum line size
* a way of splitting them (I mean, the same way across the code)

Reply:
We are splitting with back-slash in other parts of the code.

Reply:
>  We are splitting with back-slash in other parts of the code.

I wanted to change it because it forces to break indentation... but agama-i18n eslint plugin [complains](https://github.com/openSUSE/eslint-plugin-agama-i18n/blob/262dd24a52cf5c140e7263f62e32e007172b06bd/string-literals.js#L11) if  `+` is used.

@lslezak told me that 

>  unfortunately xgettext does not support _("foo" + "bar") properly, the created POT file only contains the "foo" part but not the "bar" :-/

So, I've used back-slash too.